packages/date-picker/src/FilterDateRangePicker/FilterDateRangePicker.tsx

```diff
@@ -112,16 +124,18 @@ export const FilterDateRangePicker = ({
   >(transformDateToInputValue(selectedRange?.to))
 
   useEffect(() => {
-    const rangeStart = transformDateToInputValue(selectedRange?.from)
-    const rangeEnd = transformDateToInputValue(selectedRange?.to)
-
-    if (rangeStart !== inputRangeStartValue) {
-      setInputRangeStartValue(transformDateToInputValue(selectedRange?.from))
-    }
-    if (rangeEnd !== inputRangeEndValue) {
-      setInputRangeEndValue(transformDateToInputValue(selectedRange?.to))
+    if (!isOpen) {
+      const rangeStart = transformDateToInputValue(selectedRange?.from)
+      const rangeEnd = transformDateToInputValue(selectedRange?.to)
+
+      if (rangeStart !== inputRangeStartValue) {
+        setInputRangeStartValue(transformDateToInputValue(selectedRange?.from))
```

Comment:
Can you use the `rangeStart` variable here instead of `transformDateToInputValue` again?

Reply:
I most certainly can 😂 Derp